src/interactive/plots.ts

```diff
@@ -626,12 +829,83 @@ export function displayPlot(params: { kind: string, data: string }) {
         `
 
         grid_panel.webview.html = grid_content
-    }
-    else {
+    } else {
         throw new Error()
     }
 
     if (vscode.workspace.getConfiguration('julia').get('focusPlotNavigator')) {
         g_plotNavigatorProvider?.showPlotNavigator()
     }
 }
+
+/**
+ * Send export request(message) to the plot pane.
+ */
+function requestExportPlot() {
+    g_plotPanel.webview.postMessage({
+        type: 'requestSavePlot',
+        body: { index: g_currentPlotIndex },
+    })
+}
+
+function requestCopyPlot() {
+    g_plotPanel.webview.postMessage({
+        type: 'requestCopyPlot',
+        body: { index: g_currentPlotIndex },
+    })
+}
+
+interface ExportedPlot {
+  svg?: string;
+  png?: string;
+  gif?: string;
+  index: number;
+}
+
+type FileLike = string | Buffer;
+/**
+ * Write svg file of the plot to the plots directory.
+ * @param plot
+ */
+function savePlot(plot: ExportedPlot) {
+    const plotName = `plot_${plot.index + 1}`
+
+    if (plot.svg !== null) {
+        const fileName = `${plotName}.svg`
+        _writePlotFile(fileName, plot.svg)
+    } else if (plot.png !== null) {
+        const fileName = `${plotName}.png`
+        const buffer = Buffer.from(plot.png, 'base64')
+        _writePlotFile(fileName, buffer)
+    } else if (plot.gif !== null) {
+        const fileName = `${plotName}.gif`
+        const buffer = Buffer.from(plot.gif, 'base64')
+        _writePlotFile(fileName, buffer)
+    }
```

Comment:
Would be good to add a warning message if this fails, just so the user isn't super confused as to what's going on.

Reply:
Resolved in f710ca1, and cc5586b.